src/go/k8s/cmd/configurator/main.go

```diff
@@ -283,6 +283,12 @@ func getExternalIP(node *corev1.Node) string {
 			return address.Address
 		}
 	}
+	// If no external IP is found, fallback on the internal IP. This is mainly useful for Kind.
+	for _, address := range node.Status.Addresses {
```

Comment:
oh neat, yeah that feature sounds helpful, I wonder if we should do this only if developerMode==true or we should add another bool flag like developerKindMode=true to enable this. Thoughts? @RafalKorepta @dimitriscruz

Reply:
I'd say `developerMode` is enough of an indicator.

- It'd need to be passed to the configurator similarly to 
- We have another instance of `getExternalIP` in the controller that is used for the status. That would need to be updated as well (or refactored into one function):
https://github.com/vectorizedio/redpanda/blob/5ce292b50059cffbe1f0b779e4d7355bb7e3febf/src/go/k8s/controllers/redpanda/cluster_controller.go#L440

Reply:
What would you think of adding a "preferred-address-types" setting to make this super flexible? Just like the [metrics-server](https://github.com/kubernetes-sigs/metrics-server#configuration). I could even re-use this bit: https://github.com/kubernetes-sigs/metrics-server/blob/master/pkg/utils/address_resolver.go

Reply:
@zimbatm that is very cool idea! I like that